### PR TITLE
Liydu/dps mini solution

### PR DESCRIPTION
**Pull request by ihere1**

### CODE REVIEW

File: docs/_projects/06-dps.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+2. Connect DevKit to computer and open VS Code, expand left side **ARDUINO EXAMPLES** section, select **Examples for MXCHIP AZ3166 > AzureIoT**, and click **DPS**. This will open a new VS Code window with project folder in it.
+  ![Screenshot]()
+
+3. Open **DPS.ino**, Find and replace `[Global Device Endpoing]` and `[ID Scope]` with the values you just note down.
+  ![Screenshot]()
+
+4. Use **Quick Open** in VS Code (Windows: `Ctrl+P`, macOS: `Cmd+P`) and type **task device-upload** to build and upload the code to the DevKit.
+
+5. Open serial monitor and note down DevKit firmware version.
```
[the remark this answers is not in the document]
Comment (ArthurMa1978) [reply docs/_projects/06-dps.md]: this step shall be changed, as Helen has changed the tool of generate RegistrationID

File: docs/_projects/06-dps.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+Each of two characters are used as Hex value in the security calculation. So the above sample USD is resolved to: "`0x19`, `0xe2`, `0x5a`, `0x25`, `0x9d`, `0x0c`, `0x2b`, `0xe0`, `0x3a`, `0x02`, `0xd4`, `0x16`, `0xc0`, `0x5c`, `0x48`, `0xcc`, `0xd0`, `0xcc`, `0x7d`, `0x17`, `0x43`, `0x45`, `0x8a`, `0xae`, `0x1c`, `0xb4`, `0x88`, `0xb0`, `0x74`, `0x99`, `0x3e`, `0xae`".
+
+To save Unique Device Secret on the DevKit:
+
+1. Take the sample USD and change the characters to other values between `0` and `e` to use it as your own UDS.
+
+2. With the DevKit connected to computer, hold down button A, then push and release the reset button to enter configuration mode. The screen should show the DevKit id and **'Configuration'**.
+
+3. Open serial monitor. Type **set_dps_uds [your_own_uds_value]** and press the Enter key to save it.
```
[the remark this answers is not in the document]
Comment (ArthurMa1978) [reply docs/_projects/06-dps.md]: agree, need to tell costumer to use putty or other serial port terminal do this step

File: docs/_projects/06-dps.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+## Generate X.509 certificate
+
+{% include switch.html content = page.variable %}
+
+### Windows
+
+1. Open file explorer and go to **C:\Users\\[username]\AppData\Local\Arduino15\packages\AZ3166\hardware\stm32f4\\[version]\libraries\AzureIoT\examples\DPS\\.build\\**, find and copy **DPS.ino.bin** and **DPS.ino.map**.
+
+2. Paste these two files into **C:\Users\\[username]\AppData\Local\Arduino15\packages\AZ3166\hardware\stm32f4\\[version]\tools\dice_device_enrollment\\**.
```
Comment (ArthurMa1978) [inline docs/_projects/06-dps.md]: consider to put this tool in the DPS mini-solution folder

Comment (ArthurMa1978) [reply docs/_projects/06-dps.md]: provide the link of the tool source url in git

File: docs/_projects/06-dps.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+
+3. Open serial monitor. Type **set_dps_uds [your_own_uds_value]** and press the Enter key to save it.
+
+## Generate X.509 certificate
+
+{% include switch.html content = page.variable %}
+
+### Windows
+
+1. Open file explorer and go to **C:\Users\\[username]\AppData\Local\Arduino15\packages\AZ3166\hardware\stm32f4\\[version]\libraries\AzureIoT\examples\DPS\\.build\\**, find and copy **DPS.ino.bin** and **DPS.ino.map**.
```
[the remark this answers is not in the document]
Comment (ArthurMa1978) [reply docs/_projects/06-dps.md]: this step should be run after build the DPS mini-solution

File: docs/_projects/06-dps.md [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+
+{% include toc icon="columns" %}
+
+## Before you begin
+
+To complete the steps in this tutorial, you need the following:
+
+* Prepare your DevKit with [Getting Started Guide]({{"/docs/get-started/" | absolute_url }}).
+* An IoT Hub with Device Provisioning Service instance created and linked, see [Set up auto provisioning](https://docs.microsoft.com/en-us/azure/iot-dps/quick-setup-auto-provision).
+
```
Comment (ArthurMa1978) [inline docs/_projects/06-dps.md]: tell people to upgrade the firmware

File: docs/_projects/06-dps.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (ArthurMa1978) [reply docs/_projects/06-dps.md]: not in there, it's in the folder that the new project has been created

File: docs/_projects/06-dps.md (hunk 3 above)
Comment (ArthurMa1978) [reply docs/_projects/06-dps.md]: just move this tool to the mini-solution, so copy these files to the **tool\** folder in use project

File: docs/_projects/06-dps.md [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+2. Press the **Reset** button on your DevKit.
+
+You should see the DevKit start the registration with your Device Provisioning service.
+
+## Verify the DevKit is registered on IoT Hub
+
+Once your device boots, the following actions should take place:
+
+1. The device sends a registration request to your Device Provisioning service.
```
[the remark this answers is not in the document]
Comment (liydu) [reply docs/_projects/06-dps.md]: Fixed

File: docs/_projects/06-dps.md (hunk 5 above)
Comment (liydu) [reply docs/_projects/06-dps.md]: Added.

**Pull request merged**